src/Products/PlonePAS/plugins/passwordpolicy.py

```diff
@@ -57,23 +58,15 @@ def validateUserInfo(self, user, set_id, set_info):
         password = set_info.get("password", None)
         if password is None:
             return []
-        elif password == "":
-            return [
-                {
-                    "id": "password",
-                    "error": _(
-                        "Minimum ${min_chars} characters.",
-                        mapping={"min_chars": str(self.min_chars)},
-                    ),
-                }
-            ]
         elif len(password) < self.min_chars:
             return [
                 {
                     "id": "password",
-                    "error": _(
-                        "Your password must contain at least ${min_chars} characters.",
-                        mapping={"min_chars": str(self.min_chars)},
+                    "error": translate(
+                        _(
+                            "Your password must contain at least ${min_chars} characters.",
+                            mapping={"min_chars": str(self.min_chars)},
+                        )
```

Comment:
@tschorr I think you need to pass `context=getRequest()` to translate in order for it to be able to determine the current language.

Reply:
done.